index.js

```diff
@@ -94,9 +94,11 @@ async function underPressure (fastify, opts) {
         response: {
           200: {
             type: 'object',
-            properties: {
-              status: { type: 'string' }
-            }
+            properties: Object.assign(
+              {},
+              opts.exposeStatusRoute.routeResponseSchemaOpts,
+              { status: { type: 'string' } }
```

Comment:
minor but you can avoid creating the empty object

```suggestion
              { status: { type: 'string' } },
              opts.exposeStatusRoute.routeResponseSchemaOpts
```